Makefile

```diff
@@ -129,4 +130,13 @@ docs: docs_source/**/*
 	$(MAKE) -C docs_source
 	mv docs_source/public docs
 
-.PHONY: test gotest vendor-update vendor-clean xcompile package samples/errors/*.hcl blackbox/*.sh lint bench license-check
+integration: xcompile
+	@echo
+	@echo === quick integration test ===
+	$(INTEGRATION_CMD) testing/quick-integration.hcl
+	@ if [ -n "$$CVG_FULL_INTEGRATION" ];\
```

Comment:
I think we should probably document the use of this env var somewhere. maybe in CONTRIBUTING.md?